README.md

```diff
@@ -1,46 +1,14 @@
 # Go for Visual Studio Code
 
-[![Slack](https://img.shields.io/badge/slack-gophers-green.svg?style=flat)](https://gophers.slack.com/messages/vscode/) [![Build Status](https://travis-ci.org/Microsoft/vscode-go.svg?branch=master)](https://travis-ci.org/Microsoft/vscode-go)
+[![Slack](https://img.shields.io/badge/slack-gophers-green.svg?style=flat)](https://gophers.slack.com/messages/vscode/)
 
-> Read the [We are moving 🏡 🚚 🏡](#we-are-moving---) section to learn about the new home for this extension and subscribe to [Microsoft/vscode-go#3247](https://github.com/microsoft/vscode-go/issues/3247) for updates.
-
-This extension adds rich language support for the [Go language](https://golang.org/) to VS Code.
-
-See the [Changelog](https://github.com/Microsoft/vscode-go/blob/master/CHANGELOG.md) to know what has changed over the last few versions of this extension.
-
-## Table of Contents
-
-[**We are moving!**](#we-are-moving---)
-
-- [Language Features](#language-features)
-	- [IntelliSense](#intellisense)
-	- [Code Navigation](#code-navigation)
-	- [Code Editing](#code-editing)
-	- [Diagnostics](#diagnostics)
-	- [Testing](#testing)
-	- [Debugging](#debugging)
-	- [Others](#others)
-- [How to use this extension?](#how-to-use-this-extension)
-	- [Go Language Server](#go-language-server)
-		- [Settings to control the use of the Go language server](#settings-to-control-the-use-of-the-go-language-server)
-		- [Provide feedback on gopls](#provide-feedback-on-gopls)
-	- [Linter](#linter)
-	- [Commands](#commands)
-	- [Optional: Debugging](#optional-debugging)
-		- [Remote Debugging](#remote-debugging)
-- [Install or update all dependencies](#install-or-update-all-dependencies)
-- [Building and Debugging the Extension](#building-and-debugging-the-extension)
-- [Tools this extension depends on](#tools-this-extension-depends-on)
-- [Frequently Asked Questions (FAQ)](#frequently-asked-questions-faq)
-- [Contributing](#contributing)
-- [Code of Conduct](#code-of-conduct)
-- [License](#license)
 
 ## We are moving! 🏡 🚚 🏡 
```

Comment:
"We have moved" instead

Reply:
But that will break the link 

![](https://media2.giphy.com/media/L95W4wv8nnb9K/giphy.gif)

Reply:
Which is why the first sentence under it says "As of June 2020, our new home is .."

Reply:
Oh right of course, sorry. Though I think the link will still take people to the README so maybe it's not so bad?

Reply:
Fine